x-pack/plugins/ml/public/application/jobs/new_job/pages/components/job_details_step/components/additional_section/components/calendars/calendars_selection.tsx

```diff
@@ -65,7 +72,7 @@ export const CalendarsSelection: FC = () => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [selectedCalendars.join()]);
 
-  const comboBoxProps: EuiComboBoxProps<Calendar> = {
+  const comboBoxProps: EuiComboBoxProps<MlCalendar> = {
```

Comment:
When I clone a job with a DST calendar, the calendar is also set in the (regular) calendars input. This also happens if you create a new calendar, and then go back and forward in the job wizard.

<img width="1911" alt="Screenshot 2024-10-03 at 14 11 30" src="https://github.com/user-attachments/assets/5cd71b79-ac0c-42f6-bf46-cb480d2d2528">

Probably also related - add a DST calendar, go forward in the wizard and then back, delete the regular calendar, then forward and back again, and the DST calendar is cleared.

Reply:
Fixed in https://github.com/elastic/kibana/pull/193605/commits/5b1a167d52cd3246688259ac3a26629b899709d8